### PR TITLE
[DOC] Update README

### PR DESCRIPTION
**Pull request by xf-**
# Don't merge yet

## Description

Please review and add comments to improve the README..

## Motivation and Context

Out-of-date, format could be improved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,22 +31,23 @@
 - Configurable reverse geocoding server
 - [A lot more in commit history](https://github.com/ffrgb/meshviewer/commits/develop)
 
-# Demo (embedded):
+## Demo:
 
-https://regensburg.freifunk.net/netz/karte/
+https://regensburg.freifunk.net/netz/karte/ embedded and<br>
+https://regensburg.freifunk.net/meshviewer/ standalone
```
Comment (genofire) [inline README.md]: https://map.bremen.freifunk.net/ (with a [patch](https://github.com/FreifunkBremen/meshviewer-ffrgb/pull/1) for a different `nodes.json` generated by [respond-collector](https://github.com/FreifunkBremen/respond-collector)

**Review by genofire — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+# Meshviewer
+[![Build Status](https://img.shields.io/travis/ffrgb/meshviewer.svg?style=flat-square)](https://travis-ci.org/ffrgb/meshviewer)
+[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/ffrgb/meshviewer/develop.svg?style=flat-square)](https://scrutinizer-ci.com/g/ffrgb/meshviewer/?branch=develop)
 
-### Main differences to https://github.com/ffnord/meshviewer
-#### Some features are maybe merged
+Web-app to visualize nodes and links on a map for Freifunk open mesh network.
+
+#### Main differences to https://github.com/ffnord/meshviewer<br>
+_Some features are maybe implemented/merged_
```
Comment (SimJoSt) [inline README.md]: suggestion:
> _Some similar features might have been implemented/merged_

File: README.md [hunk 3]
```diff
@@ -27,22 +31,24 @@
 - Configurable reverse geocoding server
 - [A lot more in commit history](https://github.com/ffrgb/meshviewer/commits/develop)
 
-# Demo (embedded):
+## Demo:
 
-https://regensburg.freifunk.net/netz/karte/
+https://regensburg.freifunk.net/netz/karte/ embedded and<br/>
+https://regensburg.freifunk.net/meshviewer/ standalone<br/>
+https://map.bremen.freifunk.net/ (with a [patch](https://github.com/FreifunkBremen/meshviewer-ffrgb/pull/1) for a different `nodes.json` generated by [respond-collector](https://github.com/FreifunkBremen/respond-collector)
```
Comment (SimJoSt) [inline README.md]: missing trailing `)`

File: README.md [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+#### Main differences to https://github.com/ffnord/meshviewer<br>
+_Some features are maybe implemented/merged_
 
 - Add modes - For example add a night layer and style
 - Updates selected node or list (incl. image stats cache-breaker) - not only overview tables
 - Node filter is implemented
 - Zoom level if you click a node (`nodeZoom`) - Zoom level 22 available, but it is to close for a click
 - Formatted Code
 - Translation support - https://crowdin.com/project/meshviewer - Contact us for new languages
+  - Currently avialable en, de & fr
```
Comment (SimJoSt) [inline README.md]: I would insert a `:` after `available`

File: README.md [hunk 5]
```diff
@@ -1,14 +1,19 @@
-[![Build Status](https://travis-ci.org/ffrgb/meshviewer.svg?branch=develop&style=flat-square)](https://travis-ci.org/ffrgb/meshviewer)
+# Meshviewer
+[![Build Status](https://img.shields.io/travis/ffrgb/meshviewer.svg?style=flat-square)](https://travis-ci.org/ffrgb/meshviewer)
+[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/ffrgb/meshviewer/develop.svg?style=flat-square)](https://scrutinizer-ci.com/g/ffrgb/meshviewer/?branch=develop)
 
-### Main differences to https://github.com/ffnord/meshviewer
-#### Some features are maybe merged
+Web-app to visualize nodes and links on a map for Freifunk open mesh network.
+
+#### Main differences to https://github.com/ffnord/meshviewer<br>
```
Comment (SimJoSt) [inline README.md]: trailing `<br>` not necessary

Comment (SimJoSt) [inline README.md]: As it is not possible to see, that where this repo was forked from, I would expand the description with:
> (fork origin; not maintained anymore)

Either at the end of the title or as a new line.
Maybe merge it with the current description:
>_fork origin; not maintained anymore; not actively watched, some similar features might have been implemented/merged_

File: README.md [hunk 6]
```diff
@@ -1,14 +1,19 @@
...
 
 - Add modes - For example add a night layer and style
 - Updates selected node or list (incl. image stats cache-breaker) - not only overview tables
 - Node filter is implemented
 - Zoom level if you click a node (`nodeZoom`) - Zoom level 22 available, but it is to close for a click
 - Formatted Code
 - Translation support - https://crowdin.com/project/meshviewer - Contact us for new languages
+  - Currently avialable en, de & fr
 - Grunt inline for some css and js - less requests
 - Icon font with only needed icons
```
Comment (SimJoSt) [inline README.md]: suggestion to improve English:
> Icon font with needed icons only

File: README.md [hunk 7]
```diff
@@ -1,14 +1,19 @@
...
 - Upgrade to grunt v1.x (Tested with Node.js 4 LTS,6 LTS,7 Linux,OSX,W**)
```
Comment (SimJoSt) [inline README.md]: change `grunt` to uppercase:
> Upgrade to Grunt v1.x (Tested with Node.js 4 LTS,6 LTS,7 Linux,OSX,W**)

Comment (SimJoSt) [inline README.md]: suggestion to improve English
> Moved some css and js inline

File: README.md (hunk 4 above)
Comment (SimJoSt) [inline README.md]: correct typo: `avialable` -> `available`

File: README.md (hunk 7 above)
Comment (SimJoSt) [inline README.md]: move `css` and `js` to uppercase:
> Inline some CSS and JS

File: README.md [hunk 8]
```diff
@@ -17,7 +22,6 @@
   - Load only  moment.js without languages - Languages included in translations
   - Remove unneeded components (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
 - Rbush v2 - performance boost in last versions. (Positions labels and clients on the map)
-- Right click open layermenu
 - Remove ruby dependency
 - FixedCenter is required
 - Rewrite Scss, SASS lint and variables for easy customization/adjustments
```
Comment (SimJoSt) [inline README.md]: to the best of my knowledge I would change `Scss` to `SCSS` and `SASS` to `Sass`

File: README.md [hunk 9]
```diff
@@ -17,7 +22,6 @@
   - Load only  moment.js without languages - Languages included in translations
```
Comment (SimJoSt) [inline README.md]: missing `are`:
> Languages are included in translations

File: README.md [hunk 10]
```diff
@@ -17,7 +22,6 @@
...
   - Remove unneeded components (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
 - Rbush v2 - performance boost in last versions. (Positions labels and clients on the map)
```
Comment (SimJoSt) [inline README.md]: I would change `Rbush` to `RBush` and remove the `.`

File: README.md [hunk 11]
```diff
@@ -27,22 +31,24 @@
 - Configurable reverse geocoding server
 - [A lot more in commit history](https://github.com/ffrgb/meshviewer/commits/develop)
```
Comment (SimJoSt) [inline README.md]: add `the` to improve English:
> [A lot more in the commit history](https://github.com/ffrgb/meshviewer/commits/develop)

File: README.md [hunk 12]
```diff
@@ -27,22 +31,24 @@
...
 
-- yarn
+- yarn (npm fallback)
 - grunt-cli
 
-# Installing dependencies
+### Installing dependencies
 
-### npm is still possible, but yarn is much faster https://yarnpkg.com/
+_npm is still possible, but yarn is much faster https://yarnpkg.com/_
```
Comment (SimJoSt) [inline README.md]: add `to use` to improve English:
> _npm is still possible to use, but yarn is much faster https://yarnpkg.com/_

File: README.md [hunk 13]
```diff
@@ -70,69 +76,66 @@ Use `grunt serve` for development.
 
 ## Support/Help
 
-- IRC: irc.hackint.org #freifunkRGB
+- IRC: irc.hackint.org #freifunkRGB or development-channel #meshviewer
 - Feel free to open an issue for a problem or a idea.
```
Comment (SimJoSt) [inline README.md]: change `a idea` to `an idea` to improve English:
> Feel free to open an issue for a problem or an idea.

Comment (SimJoSt) [inline README.md]: add a link to `issue`:

    Feel free to open an [issue](https://github.com/ffrgb/meshviewer/issues/new) for a problem or an idea.

File: README.md [hunk 14]
```diff
@@ -70,69 +76,66 @@ Use `grunt serve` for development.
 
 ## Support/Help
 
-- IRC: irc.hackint.org #freifunkRGB
+- IRC: irc.hackint.org #freifunkRGB or development-channel #meshviewer
```
Comment (SimJoSt) [inline README.md]: maybe a little more structure for the IRC channels and IRC links:

    - IRC on irc.hackint.org
        - [#freifunkRGB](irc://irc.hackint.org/freifunkRGB)
        - [#meshviewer](irc://irc.hackint.org/meshviewer) (development-channel)

File: README.md [hunk 15]
```diff
@@ -282,15 +285,16 @@ Example for `supportedLocale`:
...
 
 ## Sponsoring / Supporting
 - [BrowserStack](https://www.browserstack.com/) for providing a awesome testing service for hundreds of browsers
-- [Travis CI](https://travis-ci.org/) for testing every push and pull request
+- [Travis CI](https://travis-ci.org/) for building meshviewer on every push and pull request
+- [Scrutinizer CI](https://scrutinizer-ci.com/g/ffrgb/meshviewer/) for testing code quality on every push and pull request
 - [Crowdin](https://crowdin.com/) for providing easy non-developer translation environment
 
-These tools need a lot of infrastructure behind and don't charge open source software
+These tools need a lot of infrastructure and prodice a free account for open source software.
```
Comment (SimJoSt) [inline README.md]: `prodice` to `provide` to improve English

File: README.md [hunk 16]
```diff
@@ -282,15 +285,16 @@ Example for `supportedLocale`:
...
     
-## cacheBreaker (string)
+### cacheBreaker (string)
 
 Will be replaced in every build to avoid missing or outdated language strings, because language.json isn't up to date.
 
 _Fixed value (y0z)._
 
 ## Sponsoring / Supporting
 - [BrowserStack](https://www.browserstack.com/) for providing a awesome testing service for hundreds of browsers
```
Comment (SimJoSt) [inline README.md]: change `a awesome` to `an awesome` to improve English

File: README.md [hunk 17]
```diff
@@ -282,15 +285,16 @@ Example for `supportedLocale`:
...
-- [Travis CI](https://travis-ci.org/) for testing every push and pull request
+- [Travis CI](https://travis-ci.org/) for building meshviewer on every push and pull request
+- [Scrutinizer CI](https://scrutinizer-ci.com/g/ffrgb/meshviewer/) for testing code quality on every push and pull request
 - [Crowdin](https://crowdin.com/) for providing easy non-developer translation environment
```
Comment (SimJoSt) [inline README.md]: change `providing easy` to `providing an easy` to improve English

**Review by SimJoSt — CHANGES_REQUESTED** [review]
I supplied some suggestions for general improvements.

**Comment by SimJoSt** [comment]
At the moment the changed features are announced in a number of different ways:
- add
- adds
- added

If there is any interest in using just one style throughout the whole list, I will make some suggestions.

File: README.md (hunk 8 above)
Comment (xf-) [reply README.md]: i changed everything to lowercase

Comment (SimJoSt) [reply README.md]: ^^

**Pull request closed without merging**